Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+pipeline {
+  agent {
+    label 'x86&&macOS&&Apps'
+  }
+  environment {
+    VIEW = 'dsp'
+    REPO = 'lib_dsp'
+  }
+  options {
+    skipDefaultCheckout()
+  }
+  stages {
+    stage('Get view') {
+      steps {
+        prepareAppsSandbox("${VIEW}", "${REPO}")
+      }
+    }
+    stage('Library checks') {
+      steps {
+        libraryChecks("${REPO}")
+      }
+    }
+    stage('App Notes') {
+      steps {
+        buildAllApps("${REPO}/examples")
+        buildAllAppNotes("${REPO}/examples")
+      }
+    }
+    stage('Test') {
+      steps {
+        runXmostest("${REPO}", "tests") // May want to call runPytest() instead, or as well here.
```

Comment:
AFAIK all of the tests in lib_dsp currently use xmostest. I don't think that running pytest will cause any failures, but it also won't add anything yet.